packages/ckeditor5-table/src/tableutils.js

```diff
@@ -748,13 +769,15 @@ export default class TableUtils extends Plugin {
 // @param {Number} insertAt The row index of row insertion.
 // @param {Number} rows The number of rows to create.
 // @param {Number} tableCellToInsert The number of cells to insert in each row.
-function createEmptyRows( writer, table, insertAt, rows, tableCellToInsert, attributes = {} ) {
+// @param {Object} options
+// @param {Object} options.cellProperties The cell attributes.
+function createEmptyRows( writer, table, insertAt, rows, tableCellToInsert, options ) {
```

Comment:
Not sure if we gain something from making `attributes` argument more general. From what I see `createEmptyRows` is not used with anything other than just model element attributes. At the end of the passing chain we are using `createEmptyTableCell` method accepting plain attributes:
https://github.com/ckeditor/ckeditor5/blob/912cb064cd7aaad47a6d193b6b74de4193414ad5/packages/ckeditor5-table/src/utils/common.js#L35-L42

Maybe we could make it simple for now?

Reply:
Done in c7b23f2.